nix/pkgs/rusty_jwt_tools_ffi/README.md

```diff
@@ -0,0 +1,8 @@
+# How to update
+
+```
+crate2nix source add git https://github.com/wireapp/rusty-jwt-tools --rev $your-rev
+crate2nix generate
+```
+
+Currently still blocked on https://github.com/nix-community/crate2nix/issues/316.
```

Comment:
Is this a leftover? Or how do I have to understand this?

Reply:
No, this is still blocked (because it has these references to crate2nix-sources).

Reply:
Makes sense.